### DISCUSSION TITLE
404 when loading Blazorise.dll

### DISCUSSION THREAD

**Discussion by GerardFR** [Megabit/Blazorise#3305] [Q&A]
Hi,

I'm trying to include Blazorise into an existing Blazor WebAssembly project. 
As soon as I add the Blazorise Nuget package to my project and run the application, the page gets stuck on loading, and there is a 404 not found on _framework/Blazorise.dll. I also followed the get started instructions to change program.cs and index.html, but any reference to a Blazorise dll or css file generates a 404 in the network trace.

I tried to do the same with a new Blazor WebAssembly project, and it works fine.

I'm stuck, any clue on what to look for is welcome.

Gérard.

**Comment by stsrki** [comment]
Check that you have the valid `TargetFramework` in your project(s). Blazorise supports `net5.0` and `net6.0`.

**Reply by GerardFR** [reply]
Yes, the project targets net6.0. 
I can see the Blazorise.dll and Blazorise.dll.gz files in bin\Debug\net6.0\wwwroot\_framework.

**Comment by stsrki** [comment]
Did you finish all the steps in the https://blazorise.com/docs/start ?

**Reply by GerardFR** [reply]
I went through these steps once again and copy-pasted the code to be sure.
It seems to be working now, I'll test further to be sure.

Note that I added `<script src="_framework/blazor.webassembly.js"></script>` in index.html (it's not in the https://blazorise.com/docs/start)

Thanks for your help.
